GOALS.md

```diff
@@ -0,0 +1,49 @@
+# Goals
+
+## Prescriptive
+
+Push Haskell to be a better language, this will involve removing some of the historical artifacts that are remnants of the stepping stones that got us to where we are today.
```

Comment:
```suggestion
1. Push Haskell to be a better language, this will involve removing some of the historical artifacts that are remnants of the stepping stones that got us to where we are today.
```